Xamarin.Forms.Core/Items/ItemsView.cs

```diff
@@ -93,6 +93,14 @@ public int RemainingItemsThreshold
 			set => SetValue(RemainingItemsThresholdProperty, value);
 		}
 
+		public static readonly BindableProperty AreItemAnimationsEnabledProperty = BindableProperty.Create(nameof(AreItemAnimationsEnabled), typeof(bool), typeof(ItemsView), true);
+
+		public bool AreItemAnimationsEnabled
```

Comment:
please pick a better name.

here are some proposals, can't decide between the 2:
- `ItemsAnimated`
- `IsAreItemsAnimationEnabledRequestedFactory`

Reply:
Done.

Reply:
`ItemsAnimated` is not descriptive of what's happening. The items are not animated, the item _changes_ (add, remove, move, etc.) are animated. 

So maybe `ItemChangesAnimated` or `ItemChangeAnimationsEnabled`. 

(TBH, `CollectionChangeAnimationsEnabled` would be better, but possibly confusing.)

Reply:
I agree that `CollectionChangeAnimationsEnabled` would be confusing. I'll go with `ItemChangeAnimationsEnabled` for now and change it again if needed.